miso-web/src/main/webapp/scripts/list_note.js

```diff
@@ -0,0 +1,208 @@
+              Utils.ajaxWithDialog("Deleting Notes", "DELETE", url, null, function () {
+                Utils.page.pageReload();
+              });
+            });
+          },
+        },
+      ];
+    },
+    createStaticActions: function (config, entityId) {
+      return [
+        {
+          name: "Add Note",
+          handler: function () {
+            var id = config.sampleId || config.libraryId || entityId;
+            var entityType = config.entityType;
+            if (!id || !entityType) {
+              Utils.showOkDialog("Error", ["Cannot determine which entity to add a note to."]);
+              return;
+            }
+            Utils.showDialog(
+              "Create New Note",
+              "Add Note",
+              [
+                {
+                  label: "Internal Only?",
+                  property: "internalOnly",
+                  type: "checkbox",
+                  value: true,
+                },
+                {
+                  label: "Text",
+                  property: "text",
+                  type: "textarea",
+                  rows: 3,
+                  required: true,
+                },
+              ],
+              function (results) {
+                var url = "/rest/notes/" + entityType + "/" + id;
+
+                Utils.ajaxWithDialog(
+                  "Adding Note",
+                  "POST",
+                  url,
+                  {
+                    internalOnly: results.internalOnly,
+                    text: results.text,
+                  },
+                  Utils.page.pageReload
+                );
+              }
+            );
+          },
+        },
+      ];
+    },
+    createColumns: function (config, entityId) {
+      return [
+        {
+          sTitle: "Name",
+          mData: "entityName",
+          include: true,
+        },
+        {
+          sTitle: "Alias",
+          mData: "entityAlias",
+          include: true,
+        },
+        {
+          sTitle: "Note",
+          mData: "text",
+          include: true,
+        },
+        {
+          sTitle: "Owner",
+          mData: "ownerName",
+          include: true,
+        },
+        {
+          sTitle: "Created",
+          mData: "creationDate",
+          include: true,
+        },
+        {
+          sTitle: "Source",
+          mData: "source",
+          include: true,
+          mRender: function (data, type, full) {
+            if (!data) {
+              return "Current";
+            }
+            return data.charAt(0).toUpperCase() + data.slice(1);
+          },
+        },
+      ];
+    },
+    aoColumnDefs: [
+      {
+        sDefaultContent: "",
+        aTargets: ["_all"],
+      },
+    ],
+    processData: function (data) {
+      return Array.isArray(data) ? data : data.data || [];
+    },
+    bServerSide: true,
+    bStateSave: false,
+  };
+})();
+
+Utils.notes = {
```

Comment:
This is defined in `lims.js`. Overwriting it here is confusing, and since you removed `addNote`, will break the other note tables in MISO